clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -669,9 +691,17 @@ else if (value instanceof String)
             throw new ConfigException(name, value, "Not a number of type " + type);
         } catch (ClassNotFoundException e) {
             throw new ConfigException(name, value, "Class " + value + " could not be found.");
+        } catch (IOException e) {
+            throw new ConfigException(name, value, "Failed to retrieve password for " + name + " by " +
+                    "executing " + value + ". If the password is not an executable, make sure " +
+                    EXECUTABLE_PASSWORD_ENABLE_CONFIG + " is not set to True.");
         }
     }
 
+    private String execAndGetOutput(String trimmed) throws IOException {
+        return Shell.execCommand(trimmed);
```

Comment:
Here for the property, if we pass, a single script without parameters, it works.
Like ssl.truststore.password=/usr/iop/current/kafka-broker/conf/password.sh

Whereas if we provide a script with the parameter it will not work.
Like if we pass as ssl.truststore.password=/usr/iop/current/kafka-broker/conf/password.sh ssl.key.password

Here we can add a line   String[] execCmd = trimmed.split(" "); before calling ell.execCommand(execCmd);

So that user has an option he can provide a simple script or he can provide the single script to all SSL properties by passing options along with the script.
Let me know your thoughts on this.